build/Targets/Settings.props

```diff
@@ -186,7 +188,7 @@
       <PropertyGroup>
         <ProjectLanguage>C++</ProjectLanguage>
```

Comment:
It needs to be deleted but lets do it separately.  Don't want to conflate the change here unless @tmat is feeling eager.